tox.ini

```diff
@@ -18,10 +19,12 @@ setenv = VIRTUAL_ENV={envdir}
          PYFILES={toxinidir}/setup.py {toxinidir}/hotsos/ {[testenv]unit_tests}
          non-utc-tz: TZ=EST+5
          PYTHON=coverage run --source hotsos --parallel-mode
+         HOSTNAME=hostname
 deps =
     -r{toxinidir}/requirements.txt
     -r{toxinidir}/test-requirements.txt
 commands =
+    rm -rf {toxinidir}/.coverage.{env:HOSTNAME}.*
```

Comment:
We should be able to avoid this "rm" stuff. I'll submit a patch to make sure that "coverage erase" cleans these files too.